docs/ref/csrf.txt

```diff
@@ -278,8 +278,11 @@ The CSRF protection is based on the following things:
    If the :setting:`CSRF_COOKIE_DOMAIN` setting is set, the referer is compared
    against it. This setting supports subdomains. For example,
    ``CSRF_COOKIE_DOMAIN = '.example.com'`` will allow POST requests from
-   ``www.example.com`` and ``api.example.com``. If the setting is not set, then
-   the referer must match the HTTP ``Host`` header.
+   ``www.example.com`` and ``api.example.com``. The leading dot is required. It
```

Comment:
I think it might be more clear if you simply replaced "This setting supports subdomains." with "You can allow cross-subdomain requests by including a leading dot."

It doesn't seem particularly important to describe the implementation details.

Reply:
Super. Thanks Tim. Much smaller. 👍